Automattic/woocommerce-payments
# Remove fraud and risk tools feature flag again

Fixes Automattic/woocommerce-payments-server#3026

#### Changes proposed in this Pull Request

This PR removes all fraud and risk tools feature flags, it's checks, and tests related to it.









#### Testing instructions





* Remove all the fraud protection related options with:
```
npm run wp option delete wcpay_fraud_protection_settings_active
npm run wp option delete wcpay_frt_discover_banner_settings
npm run wp option delete wcpay_fraud_protection_welcome_tour_dismissed
npm run wp option delete _transient_timeout_wcpay_fraud_protection_settings
npm run wp option delete _transient_wcpay_fraud_protection_settings
```
* Run `npm run build:client` to build the new assets.
* Run `npm run listen` on your server to listen to webhooks.
* Open _Payments > Overview_, you should see the fraud and risk tools discoverability banner.
* Open _Payments > Settings_, you should see the welcome tour of fraud and risk tools.
* On the same page, you should see the fraud protection settings section, Basic protection level should be selected as default. 
* Click "Advanced" radio button, and then click the "Configure" button. You should see the  protection settings with the right filters enabled (the settings should be the defaults for high protection level).
* Change something and click "save changes". When you refresh the page, the setting you saved should be there.
* Go to your store, sell one item with a different billing country than the country you're in. 
* The order should be blocked.
* Navigate to _Payments > Transactions_ page, you should see the blocked tab. 
* When you click the blocked ones, you should see the correct transaction details.

-------------------


**Post merge**
